doc/man3/SSL_add1_expected_peer_rpk.pod

```diff
@@ -0,0 +1,93 @@
+=pod
+
+=head1 NAME
+
+SSL_get0_peer_rpk,
+SSL_rpk_send_negotiated,
+SSL_rpk_receive_negotiated,
+SSL_SESSION_get0_peer_rpk,
+SSL_add1_expected_peer_rpk - raw public key (RFC7250) support
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ EVP_PKEY *SSL_get0_peer_rpk(const SSL *s);
+ EVP_PKEY *SSL_SESSION_get0_peer_rpk(const SSL_SESSION *ss);
+ int SSL_rpk_send_negotiated(const SSL *s);
+ int SSL_rpk_receive_negotiated(const SSL *s);
+ int SSL_add1_expected_peer_rpk(SSL *s, EVP_PKEY *pkey);
+
+=head1 DESCRIPTION
+
+SSL_get0_peer_rpk() returns the peer's raw public key from SSL B<s>.
+
+SSL_rpk_send_negotiated() indicates whether sending a raw public key was
+negotiated with the peer.
+
+SSL_rpk_receive_negotiated() indicates whether receiving a raw public key
+was negotiated with the peer.
+
+SSL_SESSION_get0_peer_rpk() returns the peer's raw public key from
+SSL_SESSION B<ss> in the case of resumption.
+
+SSL_add1_expected_peer_rpk() configures the given B<pkey>
+as a public key that the peer is expected to present after negotiating
+raw public keys (RFC7250). The reference count on the provided B<pkey> is
+incremented when added to the SSL B<s>.
```

Comment:
Make it clear that this *an* acceptable public key, *one of which* the peer is expected to present.